### ISSUE TITLE
Finetuning AlexNet

### ISSUE THREAD

**Issue by mahfuj9346449** [NVIDIA/DIGITS#1742]
I am doing classification problem by finetuning AlexNet. My data set contains 20 classes. I want to use alexnet pretrained model. How to train only fc8 layer. For data set building should I build train set, val set and test set ? Thanks in advanced.

**Comment by mahfuj9346449** [comment]
I have solved the issues that I posted before. You can close this issues. Thanks.

**Comment by mahfuj9346449** [comment]
#closed
